Add testing

brucellino opened the pull request:
I've added a default scenario for testing with molecule. No tests have been added.
Also added a conditional to address #2 

Addresses #1

Commit:
Add testing

Commit:
Change the sequence of the arguments for molecule

enolfc reviewed (changes requested):
Could you rebase this on top of #4 once merged?

Commit:
Tests passing

  - Added a variable to the defaults for the packages necessary (per os)
  - Add gitignore and remove pycache file
  - Add basic tests to default scenario

Commit:
resolve conflict

brucellino commented:
I _think_ I've rebased... waiting for checks to finish.

Commit:
Re-order the steps in testing

Commit:
Clean up README and add Travis badge

enolfc commented on templates/voms.json.j2:
```diff
@@ -1,8 +1,10 @@
 {
+{% if caso_vos %}
```
if you rebased these changes should not appear here :(

brucellino replied:
I thought that was your change... Sorry man, rebases are my kryptonite.

enolfc reviewed (commented):
Let me try to rebase your changes and re-send the PR...

enolfc commented:
Moving this to #5, to have a nicer history
